tests/system_tests/f/test_initialisation.F90

```diff
@@ -0,0 +1,32 @@
+!-------------------------------------------------------------------------------
+! (c) Crown copyright 2025 Met Office. All rights reserved.
+! The file LICENCE, distributed with this code, contains details of the terms
+! under which the code may be used.
+!-------------------------------------------------------------------------------
+
```

Comment:
Might be worth a comment block to say why this is a system test rather than a unit test.
(pFUnit wanting to dictate the communicator handle.)

Reply:
Added a comment to describe the reasoning for the test.